File: Python/백준/Bronze/11050. 이항 계수 1/이항 계수 1.py
a,b=map(int,input().split())
c=1; d=1; e=1
for i in range(a):
    c*=i+1
for i in range(a-b):
    d*=i+1
for i in range(b):
    e*=i+1
print(c//(d*e))

File: C#/백준/Bronze/11050. 이항 계수 1/이항 계수 1.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aa = Console.ReadLine().Split();
            int a = int.Parse(aa[0]); int b = int.Parse(aa[1]);
            int c=1; int d=1; int e=1;
            for (int i =0; i<a; i++){
                c*=i+1;
            }
            for (int i =0; i<a-b; i++){
                d*=i+1;
            }
            for(int i=0; i<b; i++){
                e*=i+1;
            }
            Console.WriteLine(c/(d*e));
        }
    }
}